deployment/setup_admin_ssh_tunnel.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -ux
+
+source ./deployment/deployment_utils.sh
+
+setup_ssh
+
+sudo echo "127.0.0.1 dataland-admin" | sudo tee -a /etc/hosts
+ssh -L 6789:localhost:6789 -N -f ubuntu@"$TARGETSERVER_URL"
```

Comment:
lass uns die Datenbanken auch testen. Einfach ein psql command (von mir aus einfach `select 1`) in der Datenbank ausführen.

Reply:
ok